src/removeAll.js

```diff
@@ -0,0 +1,10 @@
+import { save } from './localStorage.js';
+import { render } from './renderTask.js';
+
+export default function removeAllCheckedRender() {
+  let tasks = JSON.parse(localStorage.getItem('tasks')) || [];
+  tasks = tasks.filter((task) => !task.complete);
```

Comment:
- [ ] Here as well. I would recommend using `es6` syntax as it is a project requirement that helps you in writing more clean code using ES6 syntax.